configs/components/ruby.rb

```diff
@@ -17,6 +17,11 @@
   pkg.replaces 'pe-ruby-ldap'
   pkg.replaces 'pe-rubygem-gem2rpm'
 
+  pkg.conflicts 'pe-puppet-enterprise-release', '2016.3.0.0'
```

Comment:
I'm not an expert at the syntax here, but it looks like the subject line says the conflicts should be on PE less than 2016.3 and this implies to me conflicts PE less than _or equal_ to 2016.3. Is that right?

Also, would you mind adding a code comment (maybe with ticket number) explaining why these conflicts are here?